app/src/main/java/org/oppia/app/home/UserAppHistoryViewModel.kt

```diff
@@ -8,4 +8,7 @@ class UserAppHistoryViewModel : HomeItemViewModel() {
   var isAppAlreadyOpened = ObservableField<Boolean>(false)
 
   fun setAlreadyAppOpened(alreadyOpenedApp: Boolean) = isAppAlreadyOpened.set(alreadyOpenedApp)
+  var isAppAlreadyOnboarded = ObservableField<Boolean>(false)
+
+  fun setAlreadyAppOnboarded(isalreadyOnboardedApp: Boolean) = isAppAlreadyOnboarded.set(isalreadyOnboardedApp)
```

Comment:
Check this line if it is used anywhere. else remove it.

Reply:
Removed.